src/app/app.component.ts

```diff
@@ -26,4 +38,37 @@ export class AppComponent implements OnInit {
       }
     });
   }
```

Comment:
Please move all functionality below this method to the 'titleAndMetaTagsService' service

Reply:
ok